File: 프로그래머스/1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.java
class Solution {
    public int[] solution(long n) {
        String text = String.valueOf(n);
        int[] answer = new int[text.length()];
        int index = 0;
        
        for (int i = text.length() - 1; i >= 0; i--) {
            answer[index] = Character.getNumericValue(text.charAt(i));
            index++;
        }
        return answer; 
    }
}

File: 프로그래머스/1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(long long n) {
    string str = to_string(n);
    vector<int> answer;    

    for (int i = str.length() - 1; i >= 0; i--) {
        answer.push_back(str[i] - '0');
        }
    return answer;
}   